src/app/search/components/patient_search/patient_search.html

```diff
@@ -7,12 +7,21 @@ <h2 class="panel-title">Patient Query</h2>
       </div>
       <div class="panel-body slide" ng-if="!vm.hidePanel">
         <div class="form-group">
-          <label class="col-sm-12 control-label">Name<span ng-if="vm.query.patientNames && vm.query.patientNames.length !== 1">s</span> <span class="text-danger">*</span></label>
-          <div class="col-sm-12">
-            <ai-names names="vm.query.patientNames"
-                      active-form="vm.queryForm"
-                      show-form-errors="vm.showFormErrors"
-                      submit-form="vm.searchForPatient"></ai-names>
+        	<label class="col-sm-12 control-label">Name<span class="text-danger">*</span></label>
+            	<div class="form-group" ng-class="{ 'has-error' : (vm.showFormErrors || vm.queryForm.given.$touched) && vm.queryForm.given.$error.required }">
+        			<label for="given" class="control-label col-sm-4">First <span class="text-danger">*</span></label>
+        				<div class="col-sm-8">
+          					<input class="input-sm form-control" type="text" ng-keyup="$event.keyCode == 13 && vm.submitForm()" ng-model="vm.query.patientNames[0].givenName[0]" id="given" name="given" placeholder="First" required></input>
+          					<span class="text-danger form-control-static" ng-if="(vm.showFormErrors || vm.queryForm.given.$touched) && vm.queryForm.given.$error.required">Field is required</span>
+        				</div>
+        		</div>
+      			<div class="form-group" ng-class="{ 'has-error' : (vm.showFormErrors || vm.queryForm.family.$touched) && vm.queryForm.family.$error.required }">
+        			<label for="family" class="control-label col-sm-4">Last <span class="text-danger">*</span></label>
+        				<div class="col-sm-8">
+          					<input class="input-sm form-control" type="text" ng-keyup="$event.keyCode == 13 && vm.submitForm()" ng-model="vm.query.patientNames[0].familyName" id="family" name="family" placeholder="Last" required></input>
+          					<span class="text-danger form-control-static" ng-if="(vm.showFormErrors || vm.queryForm.family.$touched) && vm.queryForm.family.$error.required">Field is required</span>
+        				</div>
+      			</div>
```

Comment:
I think the rest of this HTML looks good, but it's hard to tell without seeing it on a page. Definitely appreciate teh correct (I hope) errors, labels, and error messages